src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,228 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import exceptions.DeadlineCommandException;
+import exceptions.EventCommandException;
+import exceptions.MarkUnmarkCommandException;
+import exceptions.TodoCommandException;
+import task.Task;
+import task.Todo;
+import task.Deadline;
+import task.Event;
+import storage.Storage;
+
+public class Duke {
+    private static Storage store = new Storage();
+
+    public static void main(String[] args) throws FileNotFoundException {
+        printBanner();
+        Scanner in = new Scanner(System.in);
+        String userInput;
+        ArrayList<Task> listOfTasks = new ArrayList<>();
+        boolean isRunning = true;
+        store.createDirectory();
+        try {
+            store.readAndLoadFromFile(listOfTasks);
+        } catch (FileNotFoundException e) {
+            System.out.println("Notice: Local save not found. A new file will be created after adding tasks :)");
+        }
+
+
+        while (isRunning) {
+            userInput = in.nextLine();
+            String[] userInputSplit = userInput.split(" ");
+            try {
+                isRunning = runCommand(userInput, listOfTasks, userInputSplit);
```

Comment:
consider reassigning isRunning somewhere else, doesn't make sense for runCommand to return a boolean based on command type. usually boolean return indicates if the command executed properly